tbx/project_styleguide/templates/patterns/organisms/footer/footer.html

```diff
@@ -1,50 +1,76 @@
-{% load navigation_tags wagtailcore_tags %}
+{% load navigation_tags wagtailcore_tags wagtailimages_tags %}
 
 <footer class="footer">
-    <div class="footer__inner">
-        <div class="footer__primary-content">
-            <div class="footer__contact">
-                <h3 class="footer__heading">Get in touch about your project</h3>
-                <p class="footer__contact-desc">If you&apos;ve got a brief, we&apos;d love to see it. If you haven&apos;t, talk to us before you write it, we can help.</p>
-                {% include "patterns/molecules/contact-slim/contact-slim.html" %}
-            </div>
+    <div class="footer__logos">
+        {% with global_settings=settings.navigation.NavigationSettings %}
+            {% if global_settings.footer_logos %}
+                    <ul class="footer__logos-list">
+                        {% for item in global_settings.footer_logos %}
+                            {% with logo=item.value %}
+                                <li class="footer__logos-item">
+                                        {% with link=logo.link.0.value %}
+                                            {% if link %}
+                                                <a class="footer__logos-link" href="{{ link.url }}" alt="{{ link.link_text }}">
+                                                    {% image logo.image height-90 class="footer__logos-image" loading="lazy" alt="" %}
+                                                </a>
```

Comment:
Unfortunately assigning an alt attribute to a link won't announce its contents to a screenreader. Including the alt text in an image, which is the only content of a link, causes that alt text to be announced as the link text. We can leave the `title` of the link as it is, as this is a seperate visual tooltip that appears on hover

```suggestion
                                                <a class="footer__logos-link" href="{{ link.url }}" title="{{ link.link_text }}">
                                                    {% image logo.image height-90 class="footer__logos-image" loading="lazy" alt=link.link_text %}
                                                </a>
```

Reply:
For related information, here is some accessibility guidance on using images with links -  https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H30.html (in this case, we're implementing the second example)